build.sbt

```diff
@@ -3,7 +3,7 @@ import uk.gov.hmrc.DefaultBuildSettings
 val appName = "incorporated-entity-identification-frontend"
 
 ThisBuild / majorVersion := 1
-ThisBuild / scalaVersion := "2.13.16"
+ThisBuild / scalaVersion := "3.3.6"
 
```

Comment:
You can add

ThisBuild / scalacOptions += "-Wconf:msg=Flag.*repeatedly:s"

to prevent the repeated flag setting warnings in compile and it test.